src/main/java/io/bf2/kafka/common/Config.java

```diff
@@ -117,7 +124,8 @@ public class Config {
     public static final ConfigDef TOPIC_POLICY_CONFIG_DEF = new ConfigDef()
             .define(ENFORCED_VALUE_CONFIGS, ConfigDef.Type.LIST, DEFAULT_CONFIG_VALUE_CONFIGS, ConfigDef.Importance.MEDIUM, ENFORCED_VALUE_CONFIGS_DOC)
             .define(MUTABLE_CONFIGS, ConfigDef.Type.LIST, DEFAULT_MUTABLE_CONFIGS, ConfigDef.Importance.MEDIUM, MUTABLE_CONFIGS_DOC)
-            .define(RANGE_CONFIGS, ConfigDef.Type.LIST, DEFAULT_RANGE_CONFIGS, ConfigDef.Importance.MEDIUM, RANGE_CONFIGS_DOC);
+            .define(RANGE_CONFIGS, ConfigDef.Type.LIST, DEFAULT_RANGE_CONFIGS, ConfigDef.Importance.MEDIUM, RANGE_CONFIGS_DOC)
+            .define(TOPIC_CONFIG_POLICY_ENFORCED, ConfigDef.Type.BOOLEAN, false, ConfigDef.Importance.MEDIUM, TOPIC_CONFIG_POLICY_ENFORCED_DOC);
```

Comment:
The point of the new flag is to be a switch to the "original" create topic policy or this new one. So while a boolean flag could work I don't think it should be called `topic-config-policy-enforced`.

One other option could be to make it a string `$ALTER_CONFIG_POLICY_PREFIX.version`.
`version=original` the old behaviour
`version=MGDSTRM-8524` the behaviour from this epic
`version=MGDSTRM-????` switch to the behaviour of next version. 

But then I'm not really keen on the feature flags being long lived things.